modules/ebi-metagenomics/bwamem2/mem/tests/main.nf.test

```diff
@@ -0,0 +1,213 @@
+nextflow_process {
+
+    name "Test Process BWAMEM2_MEM"
+    script "../main.nf"
+    process "BWAMEM2_MEM"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "bwamem2"
+    tag "bwamem2/mem"
+
+    test("sarscov2 - fastq, index, false") {
+
+        setup {
+            run("BWAMEM2_INDEX") {
+                script "../../index/main.nf"
```

Comment:
can't find **index/main.nf** 
Do you have tests for bwa2 mem?

Reply:
I have added bwamem2/index. The module tests are default included by nf-core when the module is installed. We made the nf-test for paired and single-end for the decontamination subwf

Reply:
In my opinion we should be consistent and have test in one folder for all tools.

Could you send me a link to file `bwamem2/index/main.nf`?

Reply:
I am still working on the alternative index step but the bwamem2 index module is now there

Reply:
Regarding tests, I think the test folder structure is set by nf-core and has recently changed from having all tests in a single folder to having them separated. Is this correct @mberacochea?

Reply:
Yes, the nf-test should be in the same folder as the code of the module / subworkflow